modules/c2c/cloud2murano.lua

```diff
@@ -32,6 +32,7 @@ function cloud2murano.trigger(identity, event_type, payload, tags, options)
     }
 
     if handle_device2_event then
+      event.timestamp = event.timestamp*1000000
```

Comment:
please move it at line 28:

> timestamp = options.timestamp or os.time(os.date("!*t")) * 100000,

Reply:
We probably also need the same logic in line https://github.com/exosite/open-weather-product/blob/ffd09109394707e1ad13da80042ddb9bf0ebbdb9/modules/c2c/cloud2murano.lua#L100